report_thesis/src/sections/background/linear_and_regularization_models/ols.tex

```diff
@@ -0,0 +1,12 @@
+\subsubsection{Ordinary Least Squares Regression}\label{sec:ols}
```

Comment:
I can't find these definitions in the source. Can you clarify where you got it from? IIRC you said you modified some of the definitions, but just want to make sure. Esp. since it seems they define some of the things slightly differently.

Reply:
Which definitions?
If you refer to the mathematical definitions, they are from James2023AnIS and are identical.
The only difference is that I substitute the definition for RSS to simplify the equations, and I name the objective function as f_{model}{\beta} instead of leaving them unlabeled as they do in the source.

Reply:
All of them.

I have only been able to find the linear model definition:
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/f440811e-928b-460a-9e08-f08008c543ef)

And various forms of RSS:
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/04431e48-1371-462c-b4d0-391e87225a24)
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/fc4cab44-7ebf-4227-ab58-35845e9b701d)

These match the definitions you provide. Substituting and defining alternative variable names is fine, as long as the meaning is the same.

You call this Ordinary Least Squares, while the chapter presents Least Squares. They are not the same.
In this context, saying OLS for fitting a linear model is correct, but it isn't exactly what they define in the book, and I'd prefer staying strictly to the source.